packages/cosmos_ui_components/lib/components/cosmos_bottom_sheet_header.dart

```diff
@@ -42,7 +42,10 @@ class CosmosBottomSheetHeader extends StatelessWidget {
               ],
             ),
           ),
-        SizedBox(height: theme.spacingM),
+        if (isExtended)
+          Column(children: [SizedBox(height: theme.spacingM), SizedBox(height: theme.spacingL)])
```

Comment:
won't this work ?:
```suggestion
          SizedBox(height: theme.spacingM + theme.spacingL)
```

Also, if you need some sppacing that is between L and XL, you can simply rename them and include one additional, i.e:
rename:
XL -> XXL
XXL -> XXXL
add new:
XL
We'll deal with name discrepancies in Emeris later on

Reply:
aaand to optimize the code even more, you can do this:
```dart
SizedBox(height: isExteneded ? theme.spacingXL : theme.spacingXXXL)
```

Reply:
Updated